docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2196,6 +2196,43 @@ public class MyBean {
 ----
 <1> The annotation value is a locale tag string (IETF).
 
+==== Message Value and Localized File
```

Comment:
I'd use the title _"Message Templates"_...

Reply:
I hope I got ellipsis right. Thanks